pkgs/top-level/python-packages.nix

```diff
@@ -14066,11 +14066,11 @@ in modules // {
   };
 
   mock = buildPythonPackage (rec {
-    name = "mock-1.3.0";
+    name = "mock-2.0.0";
```

Comment:
been running 

```
_NIX_OPTIONS="max-jobs=6" nox-review --keep-going pr 19701
```

for several hours now, seems to download all versions of eclipse, total 100G on disk so far. Feels like I should run a hydra cluster! :)